gitapp/lib/HomePage/homepage.dart

```diff
@@ -0,0 +1,119 @@
+import 'dart:io';
+
+import 'package:flutter/material.dart';
+import 'package:gitapp/constants/constants.dart';
+import 'package:gitapp/user_details/user_details.dart';
+import 'package:gitapp/users_list/User_card.dart';
+
+import 'package:gitapp/users_list/users_list.dart';
+
+class HomePage extends StatefulWidget {
+  HomePage({Key? key, required this.title}) : super(key: key);
+
+  final String title;
+
+  @override
+  HomePageState createState() => HomePageState();
+}
+
+class HomePageState extends State<HomePage> with TickerProviderStateMixin {
+  List<UserCard> _card = <UserCard>[];
+  final UsersList users = new UsersList();
+  String name = "";
+  final TextEditingController _textController = new TextEditingController();
+  bool searching = false;
+
+  Future _getUsers(String text) async {
+    _card.clear();
+
+    if (text != "") {
+      name = text;
+      _textController.clear();
+
+      //users.fillSearchedUsers(text);
+      await users.fetchUsers(text);
+
+      if (users.searchedlist.length != 0) {
+        users.searchedlist.forEach((element) {
+          setState(() {
+            _card.insert(
+                0,
+                UserCard(
+                    user: element.username,
+                    image: element.avatarUrl,
+                    animationController: new AnimationController(
+                      duration: new Duration(milliseconds: animationtime),
+                      vsync: this,
+                    )));
+          });
+          _card[0].animationController.forward();
+        });
+      }
+    }
+  }
+
+  void dispose() {
+    for (UserCard message in _card) message.animationController.dispose();
+    super.dispose();
+  }
+
+  Widget _buildTextComposer() {
+    return new IconTheme(
+        data: new IconThemeData(color: Theme.of(context).accentColor),
```

Comment:
Please remove all "new" keyword occuernces  from the project

Reply:
solved